src/pages/registration/description_tab/vocabularies/HrcsActivityAutocomplete.tsx

```diff
@@ -0,0 +1,40 @@
+import { TextField, Typography } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { hrcsActivities } from '../../../../resources/vocabularies/hrcsActivities';
+import { getLanguageString } from '../../../../utils/translation-helpers';
+
+const StyledOptionText = styled(Typography)<{ indentations: number }>`
+  ${({ indentations }) => `
+    padding-left: ${indentations * 1.5}rem;
+    font-weight: ${indentations === 0 ? 500 : 400};
+    `}
+`;
+
+export const HrcsActivityAutocomplete = () => {
+  const { t } = useTranslation('registration');
+
+  const hrcsActivityOptions = hrcsActivities.categories
+    .map((category) => {
+      const { subcategories, ...mainCategory } = category;
+      const subCategories = category.subcategories ?? [];
+      return [mainCategory, ...subCategories];
+    })
+    .flat();
+
+  return (
+    <Autocomplete
+      id="hrcs-activities"
+      aria-labelledby="hrcs-activities-label"
```

Comment:
samme her med aria-labelledby, bør være konstant

Reply:
Autocomplete i MUI er litt teit her, siden dette ikke fikses automatisk som det gjør for `TextField` :/

Det er vel egentlig `id` vi burde dra ut som en konstant, siden `-label` er noe MUI legger til for label automatisk, basert på `id` til komponenten. Derfor må vi bare passe på at det som er før `-label` er lik `id`. Vil tro dette vil fikses i en fremtidig versjon, slik at `aria-labelledby` ikke trengs lenger, me

Reply:
blir rart å bare ta en som konstant, så tar begge